src/pyxplor/plot_categorical.py

```diff
@@ -0,0 +1,61 @@
+import pandas as pd
+
+def plot_categorical(
+        input_df: pd.DataFrame,
+        list_of_variables: list,
+        figsize: tuple = (10, 6),
```

Comment:
There might also be an issue putting an argument with default values in front of those without. So move this to the end of the argument list.

Reply:
I think everything below has a default - so hopefully should be good!